enrolment-service/enrol/enrol/urls.py

```diff
@@ -26,7 +26,7 @@
 projects_router = routers.NestedSimpleRouter(
     router, r'projects', lookup='project')
 projects_router.register(
-    r'users', views.ProjectUsersViewSet, base_name='project-users')
+    r'users', views.ProjectUsersViewSet, basename='project-users')
```

Comment:
Addresses a package API change

Reply:
Nice catch. This kind of change is infuriating lol